TestApp/update-npm-packages.sh

```diff
@@ -9,18 +9,22 @@ npm pack ../appcenter
 npm pack ../appcenter-analytics
 npm pack ../appcenter-crashes
 npm pack ../appcenter-link-scripts
-# workaround for macs on arm64
-npm install react-native-fs --save --legacy-peer-deps
 
 echo "Installing appcenter* packages..."
 npm install appcenter*.tgz
 
+echo "workaround for macs on arm64"
+npm install react-native-fs --save --legacy-peer-deps
+
 echo "Cleanup appcenter*.tgz"
 rm appcenter*.tgz
 
 echo "Installing other packages..."
 npm install
 
+echo "Applying patch to react-native package that fixes build on Xcode 12.5..."
+npx patch-package
```

Comment:
Let's add a comment with a link to the react-native issue.
Once the fix is released in stable version we can consider upgrading react-native and remove this line.